components/form/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -2503,7 +2503,7 @@ exports[`renders ./components/form/demo/validate-other.md correctly 1`] = `
             />
             <div
               class="ant-slider-track"
-              style="visibility:visible;left:0%;width:0%"
+              style="left:0%;width:0%"
```

Comment:
rebase ant-design:feature-3.3.0 for remove these change.

Reply:
OK